planet_rankings/planets.py

```diff
@@ -56,49 +56,46 @@
 # Here are the functions you need to implement to make things work.
 
 def get_user_selection(options):
-    """Present the user with options to choose from and return the selection.
-
-    Prints a "Number - Option" line for every option in options.
-    Asks the user to select an option by typing in its number.
-    Checks that the input represents a valid selection and returns it.
-    """
-
-    pass    # implement the function so that it mentions the docstring here
-
+    
+    #a list of options to choose from 
+    properties = (0,1,2,3,4,5)
+     
+    print(' 0 - Name')
+    print(' 1 - Diameter')
+    print(' 2 - Mass')
+    print(' 3 - Distance')
+    print(' 4 - Eccentricity')
+    print(' 5 - Moons')
+    
+    #ask and check for valid input and deal with the rest
+    while True: 
+        global x
+        try:
+            x = int(input('Auswahl: '))
+            if x in properties: 
+                return(x)
```

Comment:
Das hier ist der *richtige* Weg. Du lieferst `x` zurück und der Aufrufer kann den zugehörigen Wert unter einem *x-beliebigen* Namen abspeichern. In Deinem Fall macht ihn das Hauptprogramm ganz unten als `selection` verfügbar.